docs/changes/utils/13352.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fix error when supressing download progress bar while using non-default
```

Comment:
Oops, a typo.
```suggestion
Fix error when suppressing download progress bar while using non-default
```

Reply:
D'oh! That's what you get when submitting last thing on a Friday :flushed: